lib/active_job/queue_adapters/barbeque_adapter.rb

```diff
@@ -6,6 +6,12 @@ def enqueue(job)
         BarbequeAdapter.enqueue(job)
       end
 
+      def enqueue_at(job, timestamp)
+        raise NotImplementedError.new(
+          'Currently setting timestamp is not supported'
+        )
```

Comment:
↑ is duplicated with https://github.com/cookpad/barbeque_client/pull/10/files#diff-b85f6002659115f9ae6e2da6054c0ec1R27

You could `BarbequeAdapter.enqueue_at(job, timestamp)`.

Reply:
Done :bow: 9f62037